utils/audioFormats.ts

```diff
@@ -0,0 +1,56 @@
+import { browserDetector } from '~/plugins/browserDetection';
+
+/**
+ *
+ *
+ *
+ * @param {string} format
+ * @returns
+ */
+export function getSupportedAudioCodecs(format: string): boolean {
+  let typeString;
+
+  if (format === 'flac') {
+    if (browserDetector.isTv()) {
+      return true;
+    }
```

Comment:
Shouldn't browsers like chrome be included?

Reply:
Not all versions of Chrome support it: https://caniuse.com/flac

For TVs, we know they support it, but otherwise it's better to explicitly test (you never know). In addition, we never know what Chromium-based browsers (who would match as Chrome) would support.

Safety first :+1:

Reply:
Ok, sounds good to me.